packages/utils/crates/utils-misc/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "utils-misc"
+version = "0.1.0"
+authors = ["HOPR Association <tech@hoprnet.org>"]
+edition = "2021"
+description = "Rust WASM modules for hopr-utils"
+homepage = "https://hoprnet.org/"
+repository = "https://github.com/hoprnet/hoprnet"
+license = "LGPL-3.0-only"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[dependencies]
+real-base = { path = "../../../real/crates/real-base" }
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
```

Comment:
These dependencies can be also removed from `hoprd-misc` because the functions that used them were moved into `utils-misc`

Reply:
Done.